Serilog.Sinks.Network/Sinks/UDP/UDPSink.cs

```diff
@@ -29,15 +29,13 @@ public void Emit(LogEvent logEvent)
             using (var sw = new StringWriter(sb))
                 _formatter.Format(logEvent, sw);
 
-            var result = sb.ToString();
-            result = result.Replace("RenderedMessage", "message");
+            sb.Replace("RenderedMessage", "message");
 
-            _socket.Send(Encoding.UTF8.GetBytes(result));
+            _socket.Send(Encoding.UTF8.GetBytes(sb.ToString()));

```

Comment:
Unfortunately, this cannot be async because (for whatever reason) Serilog Sink interface (ILogEventSink) doesn't support async Emit() calls.